lib/reltool/test/reltool_server_SUITE.erl

```diff
@@ -2107,7 +2134,7 @@ save_config(Config) ->
 		     {app,stdlib,[{incl_cond,include},{vsn,undefined},
 				  {lib_dir,undefined}]},
 		     {boot_rel,"start_clean"},
-                     {rel,"no_dot_erlang","1.0",[]},
+                     {rel,"no_dot_erlang","1.0",[],[{load_dot_erlang,false}]},
 		     {rel,"start_clean","1.0",[]},
```

Comment:
Failed in tests: reltool_server:save_config also listens to InclDefs, so all 'rel' lines here will show load_dot_erlang.